MapboxAndroidDemo/build.gradle

```diff
@@ -43,6 +47,10 @@ dependencies {
   compile 'com.getbase:floatingactionbutton:1.10.1'
   compile 'com.squareup.picasso:picasso:2.5.2'
 
+  // firebase crash report
+  compile 'com.google.android.gms:play-services:10.0.1'
+  compile 'com.google.firebase:firebase-crash:10.0.1'
```

Comment:
I think this is redundant.
line 51 will work for all google services (firebase crash inclusive).
line 52 will work for only the crash report (single feature inclusion)